byway/src/main/webapp/destinations.js

```diff
@@ -224,15 +236,52 @@ window.onload = function(){
   });
   document.getElementById('user-input-form').addEventListener('submit', (event) => {
     event.preventDefault();
-    const formData = new  FormData(document.getElementById("user-input-form"));
-    fetch(buildUrlWithParams("/api/destinations", {tripKey}), {method: 'POST', body:formData}).then((response)=>
+    getDestinationPlaceId();
+    getStartPlaceId();
+  });
+}
+
+function getStartPlaceId(){
```

Comment:
This function is very similar to `getDestinationPlaceId()`. I suggest making a function like `getPlaceId()` or something, that takes in some params to distinguish between destination or start (ie boolean or string literal). The only difference is what you pass into the formData ("start-location" vs "destinations") and calling either updateStartDestination or updateLocations. Checking these two places within one single function prevents having two separate functions that serve nearly the same purpose

Reply:
fixed it